docs/en/enterprise-edition/content-collections/get-started/adoption-advisor.adoc

```diff
@@ -3,14 +3,15 @@
 
 // Review your adoption of Prisma Cloud, and identify the recommended tasks for your adoption journey.
 
-Adoption Advisor provides guidance through the three stages of the code to cloud application lifecycle: Code & Build, Deploy, and Run. It helps you navigate the Foundational, Intermediate, and Advanced tasks at each stage of the application lifecycle, and adopt the security capabilities at your own pace. You can also use the Adoption Advisor console to generate and manage PDF reports for adoption progress and widget data from the previous 30, 60, or 90 days.
+Adoption Advisor provides guidance through the three stages of the code to cloud application lifecycle: Code & Build, Deploy, and Run. It helps you navigate and adopt the Foundational, Intermediate, and Advanced security capabilities at each stage of the application lifecycle, at your own pace. Use the Adoption Advisor console to generate and manage PDF reports for adoption progress and widget data from the previous 30, 60, or 90 days.
 
-On your code to cloud journey the checks in the Code & Build, Deploy, and Run phases of the application, provide you with: 
+The checks in the Code & Build, Deploy, and Run phases of the application offer: 
 
 * Visibility and control within your Code & Build processes is critical to identify vulnerabilities and compliance violations before progressing to the next phase of the application’s lifecycle. 
 +
-In this phase, you are responsible for the secure operation of your cloud services. As an example, you use Infrastructure as code (IaC) templates to deploy, maintain, and remove cloud services such as VMs, storage buckets. Visibility and control at this stage enables your developers to proactively harden open source dependencies and IaC by identifying vulnerabilities, compliance violations, and secrets before they are compiled into applications or deployed as insecure cloud services.
-And you also may have Continuous Integration (CI) practices to rapidly and continuously develop, update, and maintain your cloud-based applications. The assembly and testing of your code into usable software packages are automated by CI systems such as Jenkins, CircleCI that integrate with the different code repositories and package management systems. These CI systems produce deployable artifacts, such as IaC, VM images, Docker images, Serverless Images, that are consumed by the release processes to drive frequent deployments and support agile development.
+In this phase, you are responsible for the secure operation of your cloud services. For example, you use Infrastructure as code (IaC) templates to deploy, maintain, and remove cloud services such as VMs, storage buckets. Visibility and control at this stage enables your developers to proactively harden open source dependencies and IaC by identifying vulnerabilities, compliance violations, and secrets before they are compiled into applications or deployed as insecure cloud services.
```

Comment:
"VMs, storage buckets" → Add conjunction "and" for clarity: "VMs and storage buckets."

Reply:
"proactively harden open source dependencies and IaC" → Replace with "proactively strengthen open-source dependencies and IaC".

Reply:
"compiled" can be replaced with "integrated".